### ISSUE TITLE
Broken in 6.2.19

### ISSUE THREAD

**Issue by infabo** [AOEpeople/scheduler_timeline#36]
1.0.0 seems not to be compatible with 6.2.19.
When I try to open the Scheduler-Module it shows:

``````
Fatal error: Class 'Scheduler' not found in xxx/typo3_src-6.2.19/typo3/sysext/core/Classes/Utility/GeneralUtility.php on line 4451```
``````

**Comment by tomasnorre** [comment]
@infabo I have just tested this, i cannot confirm the problem. 

I'll try to make a clean TYPO3 6.2.19 installation to try to confirm..

**Comment by tomasnorre** [comment]
I've have just tried with a completely new fresh clean install of TYPO3 6.2.19 I cannot reproduce the problem.. 

Have you tried to uninstall and install the extension?

**Comment by tomasnorre** [comment]
Sorry misunderstood you.. The scheduler module throws the error and not the scheduler_timeline module.. 

That i can confirm, when the scheduler_timeline is installed.. At least on a completely new/fresh installation, not on my "normal" dev environment, but perhaps it needs a rebuild soon :) 

Thanks for reporting., I'll dig into this asap.

**Comment by infabo** [comment]
#37

**Comment by tomasnorre** [comment]
Fixed with #37

**Comment by tomasnorre** [comment]
Thanks for you contribution
